river/Cursor.zig

```diff
@@ -315,7 +315,13 @@ fn handleButton(listener: *wl.Listener(*wlr.Pointer.event.Button), event: *wlr.P
                     self.seat.focus(null);
                 }
             },
-            .xwayland_unmanaged => assert(build_options.xwayland),
+            .xwayland_unmanaged => |xwayland_unmanaged| {
+                if (build_options.xwayland) {
+                    self.seat.setFocusRaw(.{ .xwayland_unmanaged = xwayland_unmanaged });
+                } else {
+                    unreachable;
+                }
+            },
```

Comment:
```suggestion
            .xwayland_unmanaged => |xwayland_unmanaged| {
                assert(build_options.xwayland);
                self.seat.setFocusRaw(.{ .xwayland_unmanaged = xwayland_unmanaged });
            },
```

Reply:
I had this originally, but `zig build` failed with the following error (including in the CI):

```
./river/Cursor.zig:320:64: error: expected type '*XwaylandUnmanaged', found 'void'
                self.seat.setFocusRaw(.{ .xwayland_unmanaged = xwayland_unmanaged });
```

Reply:
Ah, yeah that makes sense. Sorry for the noise.